configure.ac

```diff
@@ -386,20 +384,12 @@ case $host in
      CPPFLAGS="$CPPFLAGS -DMAC_OSX -DOBJC_OLD_DISPATCH_PROTOTYPES=0"
      OBJCXXFLAGS="$CXXFLAGS"
      ;;
+   *android*)
+     dnl make sure android stays above linux for hosts like *linux-android*
+     TARGET_OS=android
+     ;;
```

Comment:
those lines should be removed, we don't have this target right now. Could add it in another pull, but I think we'd need more work for `*-linux-android` to work?

Reply:
OK I will remove it here.